style.css

```diff
@@ -1,11 +1,175 @@
-@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+@import url("https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap");
 /* css reset */
 * {
   margin: 0;
   padding: 0;
   box-sizing: border-box;
 }
 
+:root {
+  --very-dark-grey: #1e1e1e;
+  --dark-grey: #27292a;
+  --light-grey: #666;
+  --off-white: #faf9f6;
+  --primary-color: #e75e8d;
+}
+
 body {
-  font-family: 'Poppins', sans-serif;
+  font-family: "Poppins", sans-serif;
+  background: var(--very-dark-grey);
+  padding: 15px 50px;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 1280px;
+  min-height: 80px;
+  margin: 0 auto;
+  padding: 15px 0;
+}
+
+#header__left {
+  display: flex;
+  align-items: center;
+}
+
+/* #header__left {
+} */
+
+#logo {
+  padding-right: 20%;
+  border-right: 1px solid var(--light-grey);
+}
+
+#search-input {
+  margin-left: 15%;
+  position: relative;
+}
+
+#search-text {
+  background-color: var(--dark-grey);
+  height: 46px;
+  border-radius: 23px;
+  border: none;
+  outline: none;
+  color: var(--off-white);
+  font-size: 15px;
+  padding: 0px 15px 0px 45px;
+}
+
+#search-input > i {
+  position: absolute;
+  color: var(--light-grey);
+  left: 20px;
+  top: 16px;
+  width: 18px;
+  height: 18px;
+  font-size: 16px;
+}
+
+header > i {
+  display: none;
+  font-weight: 900;
+  font-size: 25px;
+  color: var(--primary-color);
+}
+
+#header__right > ul {
+  display: flex;
+  align-items: center;
+  list-style: none;
+  column-gap: 30px;
+  font-size: 15px;
+}
+
+.nav-link {
+  text-decoration: none;
+  color: var(--light-grey);
+}
+
+.nav-link:hover {
+  color: var(--primary-color);
+}
+
+.nav-link.active-nav {
+  color: var(--primary-color);
+}
+
+#profile-tab {
+  display: flex;
+  align-items: center;
+  background: var(--dark-grey);
+  padding: 8px 10px 8px 20px;
+  border-radius: 23px;
+}
+
+#profile-tab img {
+  max-width: 30px;
+  border-radius: 50%;
+  margin-left: 5px;
+}
+
+.mobile {
+  display: none;
+}
+
+@media screen and (max-width: 1200px) {
+  body {
+    padding: 15px 30px;
+  }
+  #logo {
+    padding-right: 10%;
+    border: none;
+  }
+  #search-input {
+    margin-left: 0;
+  }
+}
+
+@media screen and (max-width: 992px) {
+  header {
+    position: relative;
+  }
+  header > i {
+    display: inline;
+  }
+  .mobile {
+    display: unset;
```

Comment:
unset ?

Reply:
it will change the display to the default value.